example/next.config.js

```diff
@@ -6,9 +6,13 @@ const intlRouter = new IntlRouter('pages', locales);
 module.exports = {
   i18n: {
     locales,
-    defaultLocale: 'en-CA'
+    defaultLocale: 'en-CA',
+    localeDetection: false,
```

Comment:
Is this necessary to use the module? It might be a good idea to add comments on these settings.

Reply:
Yes it's needed. In the readme I'll add some doc regarding the use case and why we need it.
Thanks